ecl/eclcmd/eclcmd_shell.cpp

```diff
@@ -205,24 +205,25 @@ void EclCMDShell::usage()
     fprintf(stdout,"\nUsage:\n"
         "    ecl [--version] <command> [<args>]\n\n"
            "Commonly used commands:\n"
-           "   deploy      create a workunit from an ecl file, archive, or dll\n"
-           "   publish     add a workunit to a query set\n"
-           "   unpublish   remove a query from a query set\n"
-           "   run         run the given ecl file, archive, dll, wuid, or query\n"
-           "   results     retrieve the results for an existing workunit\n"
-           "   activate    activate a published query\n"
-           "   deactivate  deactivate the given query alias name\n"
-           "   queries     show or manipulate queries and querysets\n"
-           "   packagemap  manage HPCC packagemaps\n"
-           "   bundle      manage ECL bundles\n"
-           "   roxie       commands specific to roxie clusters\n"
-           "   abort       abort workunit(s) for WUID or job name\n"
-           "   status      show workunit(s) current status for WUID or job name\n"
-           "   getname     provide job name from WUID\n"
-           "   getwuid     provide WUID from job name\n"
-           "   zapgen      provide ZAP file from WUID\n"
-           "   sign        add digital signature to an ecl file or a text file\n"
-           "   listkeyuid  lists all the key user IDs that can be used in the sign command\n"
+           "   deploy           create a workunit from an ecl file, archive, or dll\n"
+           "   publish          add a workunit to a query set\n"
+           "   unpublish        remove a query from a query set\n"
+           "   run              run the given ecl file, archive, dll, wuid, or query\n"
+           "   results          retrieve the results for an existing workunit\n"
+           "   activate         activate a published query\n"
+           "   deactivate       deactivate the given query alias name\n"
+           "   queries          show or manipulate queries and querysets\n"
+           "   packagemap       manage HPCC packagemaps\n"
+           "   bundle           manage ECL bundles\n"
+           "   roxie            commands specific to roxie clusters\n"
+           "   abort            abort workunit(s) for WUID or job name\n"
+           "   status           show workunit(s) current status for WUID or job name\n"
+           "   getname          provide job name from WUID\n"
+           "   getwuid          provide WUID from job name\n"
+           "   zapgen           provide ZAP file from WUID\n"
+           "   sign             add digital signature to an ecl file or a text file\n"
+           "   listkeyuid       lists all the key user IDs that can be used in the sign command\n"
+           "   url-secret-name  Generate a secret name from a url for automatic url mapping"
```

Comment:
@jakesmith Looks great, but missing newline at end which gives us a blank line before the "Run" line below.

Reply:
@afishbeck - whoops yes, well spotted, now added (squashed in)